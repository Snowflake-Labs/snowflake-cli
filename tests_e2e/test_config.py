import subprocess

import pytest


@pytest.mark.e2e
def test_config_file_creation(snowcli, test_root_path, snowflake_home, snapshot):
    output1 = subprocess.check_output([snowcli, "connection", "list"], encoding="utf-8")
    snapshot.assert_match(output1)

    output2 = subprocess.check_output([snowcli, "connection", "list"], encoding="utf-8")
    snapshot.assert_match(output2)
