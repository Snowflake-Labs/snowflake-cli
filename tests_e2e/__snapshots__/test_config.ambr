# serializer version: 1
# name: test_config_file_creation
  '''
  No data
  
  
  '''
# ---
# name: test_config_file_creation.1
  '''
  No data
  
  
  '''
# ---
